danielMunteanu/src/main/java/firstpart/testclas.java

```diff
@@ -0,0 +1,24 @@
+package firstpart;
+
+
+class testclas {
```

Comment:
let' rename this class into a more specific one

Reply:
renamed